src/daemon/rpm-ostree-fix-shadow-mode.service

```diff
@@ -7,6 +7,7 @@ Documentation=https://github.com/coreos/rpm-ostree-ghsa-2m76-cwhg-7wv6
 # the old /etc/.rpm-ostree-shadow-mode-fixed.stamp
 ConditionPathExists=!/etc/.rpm-ostree-shadow-mode-fixed2.stamp
 ConditionPathExists=/run/ostree-booted
+ConditionKernelCommandLine=ostree
```

Comment:
A comment on this would be good. E.g.:

```
# Filter out non-traditional ostree setups (e.g. live boots)
```

This is related to the lack of integration between the live CoreOS stuff and ostree (xref: https://github.com/ostreedev/ostree/issues/1921, https://github.com/ostreedev/ostree/pull/1995).